modules/social_features/social_core/social_core.install

```diff
@@ -223,6 +223,19 @@ function _social_core_create_menu_links() {
 
 }
 
+/**
+ * Implements hook_update_dependencies().
+ */
+function social_core_update_dependencies(): array {
+  // Update 123003 fixes a view corruption from 8107,
+  // so that one needs to be executed first.
+  $dependencies['social_core'][123003] = [
+    'message' => 8107,
```

Comment:
https://git.drupalcode.org/project/message/-/blob/8.x-1.x/message.install?ref_type=heads#L52

Isn't it 8105? 🙈